listener/upload_listener.py

```diff
@@ -28,6 +28,7 @@
 WORKER_THREADS = int(os.getenv('WORKER_THREADS', '30'))
 MAX_QUEUE_SIZE = int(os.getenv('MAX_QUEUE_SIZE', '30'))
 
+DISABLE_OPTIMISATION = int(os.getenv('DISABLE_OPTIMISATION', ''))
```

Comment:
use strtobool function to parse string, also specify default value, similar function in manager service:

`strtobool(os.getenv('MANAGER_STRICT_VALIDATION', 'FALSE'))`